### DISCUSSION TITLE
SMS gateway

### DISCUSSION THREAD

**Discussion by gow1998** [Sunbird-Lern/Community#118] [Query] [answered]
Hi, 
   In sunbird sms gateway instead of msg_91,  any other alternate way you have for OTP deliver through SMS?

**Comment by Jayaprakash8887** [comment]
Sunbird is tested with NIC SMS Gateway

**Reply by Jayaprakash8887** [reply]
Apart from above 2, no more.

**Reply by gow1998** [reply]
@Jayaprakash8887  In NIC SMS gateway, is it using username and password?

**Reply by Jayaprakash8887** [reply]
Yes

**Reply by gow1998** [reply]
@Jayaprakash8887  ok, how to adopt this NIC SMS gateway in sunbird setup?

**Reply by Jayaprakash8887** [reply]
@gow1998 NIC (national Informatics Center) SMS gateway is owned by Government. You have to get an account created with them and provide the necessary authentication details in the notification service and notification flink job.

**Reply by gow1998** [reply]
can you please give what are the variables need to define in the notification service and notification flink job for NIC SMS gateway? @Jayaprakash8887

**Reply by Jayaprakash8887** [reply, marked as answer]
https://lern.sunbird.org/learn/product-and-developer-guide/batch-service/datapipeline-flink-jobs#notification-job
